add docs on query page (#7066)
fishtown-analytics/docs.getdbt.com: website/docs/docs/collaborate/query-page.md

---
title: "About the Query page"
description: "Learn how to query data and perform exploratory data analysis using the Query page"
sidebar_label: "About the Query page"
tags: [Semantic Layer]
pagination_next:  "docs/collaborate/navigate-query-page"
image: /img/docs/collaborate/query-page/qp-results.jpg
---

# About the Query page <Lifecycle status="beta" />

<IntroText>
Learn how to query data with the Query page and view documentation in dbt Explorer.
</IntroText>

:::tip

> Available in private beta. To join, please reach out to your account manager. To provide feedback, use this feedback form…

Reviewer: ```suggestion
Available in private beta. To join, please reach out to your account manager. 
```

:::

> The Query page in dbt Cloud empowers users to seamlessly explore and query data with an intuitive, context-rich interface. It bridges technical and business users by combining metadata, documentation, and powerful querying capabilities &mdash; into one unified experience. 

Reviewer: ```suggestion
The Query page in dbt Cloud empowers users to seamlessly explore and query data with an intuitive, context-rich interface. It bridges technical and business users by combining metadata, documentation, and powerful querying capabilities into one unified experience. 
```

Query page in dbt Cloud integrates with [dbt Explorer](/docs/collaborate/explore-projects), [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud), [Visual Editor](/docs/cloud/visual-editor), and [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl) to make it easier for you to perform exploratory data analysis, make faster decisions, and collaborate across teams.

<Lightbox src="/img/docs/collaborate/query-page/qp-main.gif" title="Overview of the Query page and its features" />

## Key benefits

Key benefits include:

> - Quickly write, run, and iterate on SQL queries with tools like code coloring, tabbed editors, and query history.

Reviewer: ```suggestion
- Quickly write, run, and iterate on SQL queries with tools like syntax highlighting, tabbed editors, and query history.
```

> - Leverage dbt-generated metadata, tests, trust signals, and lineage for informed query construction.

Reviewer: ```suggestion
- Leverage dbt metadata, trust signals, and lineage from dbt Explorer for informed query construction.
```

- Make data accessible to users of varied technical skill levels with SQL, Semantic Layer queries, and visual tools.

Some example use cases include:
- Analysts can quickly construct queries to analyze sales performance metrics across regions and view results.

> - Engineers can use metadata to debug lineage or validate model configurations.

Reviewer: ```suggestion
```

Reviewer: this is not really a use case today!

> - All users have a rich development experience by being able to explore data leverage dbt Explorer and dbt Cloud to have an end-to-end exploration experience.

Reviewer: ```suggestion
- All users have a rich development experience powered by dbt Explorer's end-to-end exploration experience.
```

> DO WE WANT TO HAVE A 'HOW IT WORKS' SECTION SO WE CAN EXPLAIN ITS ENGINE?

Reviewer: someday yes! let's remove this for now

## Prerequisites 

- Have a dbt Cloud [Team or Enterprise](https://www.getdbt.com/pricing) account.
- Available on all [tenant](/docs/cloud/about-cloud/tenancy) configurations. Single-tenant accounts should contact your account representative for setup.
- Have a dbt Cloud [developer license](/docs/cloud/manage-access/seats-and-users) with access to the Query page.
- Set up [developer credentials](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud#get-started-with-the-cloud-ide)
- Your production and development [environments](/docs/dbt-cloud-environments) are on dbt Cloud’s ‘Latest’ [release track](/docs/dbt-versions/cloud-release-tracks) or a supported dbt version.
- Use Snowflake, BigQuery, Databricks, or Redshift.
	- Note that the dbt Semantic Layer doesn't support SSH tunneling for Postgres or Redshift connections. It also doesn't support using Single sign-on (SSO) for production credentials, though SSO is supported for development user accounts.

> - Have a successful run in the environment where you configure the [Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl).

Reviewer: we should add a bullet point for configuring the SL itself too!
